dockerfiles/str/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:20.04
+
+RUN apt-get update && DEBIAN_FRONTEND="noninteractive" apt-get install -qqy \
+    python3 \
+    python3-pip \
+    python \
+    python-dev \
+    awscli \
+    build-essential \
+    git \
+    libbz2-dev \
+    liblzma-dev \
+    make \
+    pkg-config \
+    wget \
+    unzip \
+    zlib1g-dev
+
+RUN pip3 install pybedtools pyvcf scipy numpy
+
+# Install samtools (needed to index reference fasta files)
+RUN wget -O samtools-1.9.tar.bz2 https://github.com/samtools/samtools/releases/download/1.9/samtools-1.9.tar.bz2
+RUN tar -xjf samtools-1.9.tar.bz2
+WORKDIR samtools-1.9
+RUN ./configure --without-curses && make && make install
+WORKDIR ..
```

Comment:
Is there an advantage to using `WORKDIR` rather than `cd` inside of `RUN`? I remember reading that combining multiple steps into single `RUN` statements can help reduce image size, although I'm unsure if that's true if you flatten the image at the end.

Reply:
Good point. I guess using `WORKDIR` over `cd` is suggested in [docker docs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#workdir) for clarity. Also, I am not sure about image size either. I gathered multiple `RUN`s in a single `RUN`, it had very little impact on the _compressed_ size at least: [`683.6 MB` for multiple `RUN`s vs `683.71 MB` for single `RUN`](https://hub.docker.com/repository/registry-1.docker.io/vjalili/str/tags?page=1&ordering=last_updated).